spec/unit/puppet/type/windowsfeature/windowsfeature_spec.rb

```diff
@@ -3,16 +3,18 @@
 require 'spec_helper'
 
 describe Puppet::Type.type(:windowsfeature) do
-  let :provider_class do
-    described_class.provide(:fake) { mk_resource_methods }
-  end
-  let(:provider) { provider_class.new }
   before do
-    described_class.stubs(:defaultprovider).returns provider_class
-    described_class.stubs(:provider).returns provider
+    @class = described_class
+    @provider_class = @class.provide(:fake) { mk_resource_methods }
```

Comment:
Ok, I've refactored it in such a way that Rubocop is happy and the specs are green 😄